lib/clevertap_plugin.dart

```diff
@@ -215,100 +214,129 @@ class CleverTapPlugin {
     }
   }
 
+  /**
+   * Flushes any finished events on clevertap sdk side, maybe the client attaches
+   * listener at a later point and sdk has already tried to provide callback and
+   * failed
+   */
+  void invokeStartEmission(String name) {
+    _dartToNativeMethodChannel.invokeMethod('startEmission', name);
```

Comment:
Flushing/disabling buffering for events only when a listener is set leads to these problematic outcomes:
* Events will be buffered indefinitely for all events that do not have a defined listener, leading to redundant memory usage.
* Delivering old events when a listener is set later in the life-cycle of the app. The buffering of events is helpful only with events that are triggered around/on the App Launched event. If a client sets a listener for an event on a screen that is shown for example after 1 minute of app usage, it would be a confusing behavior to trigger the event that should have occurred on/around App Launched.

We had those points in mind when we did the Unity buffers and we decided to have a 5 seconds delay after App Launched after which we clear and disable all buffers.

Reply:
added the 5 seconds change in new commit in CleverTapPlugin.java